pkgs/top-level/all-packages.nix

```
@@ -7632,6 +7632,10 @@ with pkgs;
 
   dub = callPackage ../development/tools/build-managers/dub { };
 
+  buildDubPackage = callPackage ../build-support/dlang/builddubpackage { };
+  makeDubDep = callPackage ../build-support/dlang/makedubdep { };
+  dub-to-nix = callPackage ../build-support/dlang/dub-to-nix { };
+
```

The `pkgs/by-name` failure is a bit unfortunate, because only `dub-to-nix` of these is a package, and as such would have to be in `pkgs/by-name`.

I also recommend creating an attribute set for these instead:

```suggestion
  dubSupport = callPackages ../build-support/dlang/dub-support.nix { };

```

---

For now I just made it use `inherit (callPackages ...) ...` so that it doesn't complain